README.md

```diff
@@ -115,6 +117,10 @@ const method = new Method({
 });
 ```
 
+#### Skipping argument validation
+
+If your method does not need argument validation, perhaps because it does not take any arguments, you can use `validate: true` to skip argument validation.
```

Comment:
I'd prefer `validate: null`? We can detect that it's `null` and not `undefined`, and I'd say `null` is clearer than `true` in terms of meaning "no validation"?